ops/terraform/aws/network/main.tf

```diff
@@ -0,0 +1,190 @@
+provider "aws" {
+  region = "${var.region}"
+}
+
+# VPC
+resource "aws_vpc" "application_vpc" {
+  cidr_block           = "${var.vpc_cidr_block}"
+  instance_tenancy     = "dedicated"
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+
+  tags {
+    Name = "${var.application_name}: ${var.environment} vpc"
+    env  = "${var.environment}"
+  }
+}
+
+# Subnets
```

Comment:
I think we might just want to have a single subnet. The Soapbox model is for a single application to be deployed and serving traffic -- it's sort of a BYO database situation. So I'm not sure if a separate data subnet is needed. But let me know if I misunderstood.